README.md

```diff
@@ -1076,9 +1129,9 @@ License and Author
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
         http://www.apache.org/licenses/LICENSE-2.0
-    
```

Comment:
fixing it on my next commit with the style changes :(
my lovely editor removes all white space